Sources/App/Views/Search/Search+View.swift

```diff
@@ -0,0 +1,37 @@
+import Plot
+
+
+extension Search {
+
+    class View: PublicPage {
+
+        let model: Model
+
+        init(path: String, model: Model) {
+            self.model = model
+            super.init(path: path)
+        }
+
+        override func content() -> Node<HTML.BodyContext> {
+            .group(
+                .h2(.text("Results for \(model.query)")),
+                .ul(
+                    .class("list"),
```

Comment:
Don't think this class actually exists but `package_list` was rending the two column layout. Just a placeholder.